packages/components/tree-select/tree-select.component.ts

```diff
@@ -510,6 +520,16 @@ export class KbqTreeSelect
      * If set to null or an empty string, the panel will grow to match the longest option's text.
      */
     @Input() panelWidth: KbqTreeSelectPanelWidth = this.defaultOptions?.panelWidth || null;
+    /**
+     * Controls when the search functionality is displayed based on the number of available options.
+     *
+     * Automatically enables search hiding if value provided, even if `defaultOptions.minOptionsThreshold` is provided.
+     * @default 10 or undefined
+     */
+    @Input() minOptionsThreshold =
```

Comment:
мб обозвать атрибут по аналогии с константой?
```suggestion
    @Input() searchMinOptionsThreshold =
```

Reply:
да, сделал